packages/mongodb-rag-core/src/findContent/FindContentFunc.ts

```diff
@@ -1,8 +1,15 @@
 import { EmbeddedContent } from "../contentStore/EmbeddedContent";
 import { WithScore } from "../VectorStore";
 
+type VectorSearchFilters = {
+  sourceName?: string;
+  "metadata.version.isCurrent"?: boolean;
+  "metadata.version.label"?: string;
+};
```

Comment:
can you rename these to be more semantic? 

something like:

```suggestion
type VectorSearchFilters = {
  sourceName?: string;
  version?:  {
     isCurrent?: boolean;
     label?: string;
  }
};
```

then parse it to the mdb query filters in the implementation. this way we better adhere to the principle of separating interface and implementation